apidoc/Titanium/Android/Notification.yml

```diff
@@ -1,36 +1,104 @@
 ---
 name: Titanium.Android.Notification
-summary: | 
-    The Titanium binding of an 
-    [Android Notification](http://developer.android.com/reference/android/app/Notification.html).
+summary: UI notifications that can be sent while the application is in the background.
```

Comment:
:+1: on the new summaries here and elsewhere (i.e. describing what something does rather than just as a binding for this other thing).